File: 2219-MaximumNumberOfWordsFoundInSentences/2219-MaximumNumberOfWordsFoundInSentences.py
# Last updated: 7/14/2026, 9:55:45 AM
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max=0
        for s in sentences:
            words=len(s.split())
            if words>max:
                max=words
        return max

File: 2219-MaximumNumberOfWordsFoundInSentences/2219-MaximumNumberOfWordsFoundInSentences.java
// Last updated: 7/14/2026, 9:55:51 AM
class Solution {
    public int mostWordsFound(String[] sentences) {
        int maxWords = 0;
        for (String s : sentences) {
            int count = s.split(" ").length;
            if (count > maxWords) {
                maxWords = count;
            }
        }
        return maxWords;
    }
}